### PR TITLE
feat: bump icons version

### PR DESCRIPTION
**Pull request by D7Torres**
As there are new icons available that we can use to replace font awesome.

### CODE REVIEW

**Review by hollg — APPROVED** [review]

File: lib/package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
     "yargs": "^16.2.0"
   },
   "peerDependencies": {
-    "@atom-learning/icons": "^1.0.0",
+    "@atom-learning/icons": "^1.16.0",
```
[the remark this answers is not in the document]
Comment (thomasdigby) [reply lib/package.json:123]: This version is the minimum required version for _this_ package, so if we're using new icons in `@atom-learning/components` for this update then we should bump it, if not, then it's probably best to leave it at the minimum version we need.

The version of `@atom-learning/icons` that we need in the main app is defined there as a dependency, and I think it's up to date with the latest (bar this update obvs)

[the remark this answers is not in the document]
Comment (D7Torres) [reply lib/package.json]: Cool, I reverted it, thank you

[the remark this answers is not in the document]
Comment (D7Torres) [reply lib/package.json]: I guess I should close this PR, then... although I don't get why this PR has `"@atom-learning/icons"` as peer dependency but also as dependency

[the remark this answers is not in the document]
Comment (thomasdigby) [reply lib/package.json]: It has it as a `devDependency` for tests (I think), but not as a `dependency` 👍

[the remark this answers is not in the document]
Comment (D7Torres) [reply lib/package.json]: My bad, thank you. I'll close the PR

**Comment by D7Torres** [comment]
Closed as the icons dependency it's in `atom-core` (not imported via components)

In `components` there is
* a peer dependency: The components need some icons. But not the new icons, so we don't need to update it.
* a dev dependency: The tests need some icons. But not the new icons, so we don't need to update it.

**Comment by D7Torres** [comment]
Re-opened because the new icons should actually be used in the documentation site

File: lib/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (D7Torres) [reply lib/package.json]: @thomasdigby sorry for the back and forth. But I realised the new version with the new icons it's actually needed in `components`: on the documentation site. We have a page that gets all the icons and presents them so you can browse and search by name.

So I left peer dependency and dev dependency untouched, but updated the one in `documentation/package.json`

Please let me know when you read this, and if there are no concerns I'll merge

**Review by thomasdigby — APPROVED** [review]

**Pull request merged**